### ISSUE TITLE
docs: plugins: autosklearn: Examples not present in docs

### ISSUE THREAD

**Issue by pdxjohnny** [intel/dffml#905]

**Comment by nitesh585** [comment]
I didn't found the autosklearn in docs/plugins. Can you please share the link?
I am highly interested to contribute to this organization.

**Comment by nitesh585** [comment]
@pdxjohnny Can I work on it?

**Comment by pdxjohnny** [comment]
Yes please do! That would be great! Sorry I forgot to say you don't need to ask beforehand, just comment saying you intended to work on an issue.

For this one, I had started but ran into an issue where the example was taking about an hour to run. Not sure why, here's the branch: https://github.com/pdxjohnny/dffml/commits/autosklearn_docs

Here's the CI run in question: https://github.com/pdxjohnny/dffml/runs/1444266528

```

...

Running ['python', 'run.py']

/usr/share/miniconda/lib/python3.7/site-packages/pyparsing.py:3190: FutureWarning: Possible set intersection at position 3
  self.re = re.compile(self.reString)
Accuracy: 0.9986940730779597
{'Feature1': 0.57, 'Feature2': 0.84, 'TARGET': 3.5971563942730427}
ok

----------------------------------------------------------------------
Ran 2 tests in 3860.294s

OK
```

**Comment by nitesh585** [comment]
@pdxjohnny 
found the reason why the CI test is running for 1 hour because of the last test (`python run.py`)  in the docstring test.
```
AutoSklearnConfig = make_config_numpy(
    "AutoSklearnConfig",
    autosklearn.estimators.AutoSklearnEstimator.__init__,
    properties={
        "features": (Features, field("Features to train on")),
        "predict": (Feature, field("Label or the value to be predicted")),
        "directory": (
            pathlib.Path,
            field("Directory where state should be saved",),
        ),
    },
)
```
As in the above code snippet(_model/autosklearn/dffml_model_autosklearn/config.py_), we just initialize the model with their default values.

In the docs of autosklearn(link is in below), default value of _time_left_for_this_task=3600_. That's why it is running for 1 hour.
https://automl.github.io/auto-sklearn/master/_modules/autosklearn/estimators.html

**Solution**: If somehow we can set a default value of _time_left_for_this_task_ to another value in make_config_numpy().

**Comment by pdxjohnny** [comment]
Closed via cb4914b2921d83b55e2835c5f00f2df83e517967
